marlowe-playground-client/src/HaskellEditor.purs

```diff
@@ -85,8 +85,9 @@ haskellEditor state = slot _haskellEditorSlot unit component unit (Just <<< Hask
 
 bottomPanel :: forall p. FrontendState -> HTML p HAction
 bottomPanel state =
-  div [ classes (analysisPanel state) ]
-    [ div [ classes (footerPanelBg (state ^. _showBottomPanel) HaskellEditor <> isActiveTab state HaskellEditor) ]
+  div ([ classes [ analysisPanel ] ] <> [ bottomPanelHeight (state ^. _showBottomPanel) ])
```

Comment:
I'm unclear why you're joining two static lists instead of just having one. I mean, I know monoids are awesome and everything, but they don't grow on trees(1).

1: [Proof](https://www.stackage.org/haddock/lts-16.4/containers-0.6.2.1/Data-Tree.html#t:Tree).